hub/apps/how-tos/uno-multiplatform.md

```diff
@@ -0,0 +1,142 @@
+- .NET desktop development installed (for Gtk, Wpf, and Linux Framebuffer development)
+
+## Finalize your environment
+
+1. Open a command-line prompt, Windows Terminal if you have it installed, or else Command Prompt or Windows Powershell from the Start Menu.
+
+2. Install the `uno-check` tool:
+    - Use the following command:
+
+        `dotnet tool install -g uno.check`
+
+    - To update the tool, if you already have an existing one:
+
+        `dotnet tool update -g uno.check`
+
+3. Run the tool with the following command:
+
+    `uno-check`
+
+4. Follow the instructions indicated by the tool.
+
+## Install the Uno Platform solution templates
+
+Launch Visual Studio 2022, then click `Continue without code`. Click `Extensions` -> `Manage Extensions` from the Menu Bar.
+
+:::image type="content" source="images/hello-world/manage-extensions.png" alt-text="Visual Studio Menu bar item that reads manage extensions":::
+
+In the Extension Manager expand the **Online** node and search for `Uno`, install the `Uno Platform` extension or download it from the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=unoplatform.uno-platform-addin-2022), then restart Visual Studio.
+
+:::image type="content" source="images/hello-world/uno-extensions.PNG" alt-text="Manage Extensions window in Visual Studio with Uno Platform extension as a search result":::
+
+## Create an application
+
+Now that we are ready to create a multi-platform application, the approach we'll take is to create a new Uno Platform application. We will copy XAML code from the previous tutorial's Hello World WinUI 3 project into our multi-platform project. This is possible because Uno Platform lets you reuse your existing codebase. For features dependent on OS APIs provided by each platform, you can easily make them work over time. This approach is especially useful if you have an existing application that you want to port to other platforms.
+
+Soon enough, you will be able to reap the benefits of this approach, as you can target more platforms with a familiar XAML flavor and the codebase you already have.
+
+Open Visual Studio and create a new project via `File` > `New` > `Project`:
+
+:::image type="content" source="images/hello-world/create-project.png" alt-text="Create a new project":::
+
+Search for Uno and select the Uno Platform App project template:
+
+:::image type="content" source="images/hello-world/vsix.png" alt-text="Blank, packaged WinUI 3 C# desktop app":::
+
+Specify a project name, solution name, and directory. In this example, our Hello World MultiPlatform project belongs to a Hello World MultiPlatform solution, which will live in C:\Projects:
+
+:::image type="content" source="images/hello-world/configure-project.png" alt-text="Specify project details":::
+
+Create a new C# solution using the **Uno Platform App** type from Visual Studio's **Start Page**. You may need to choose a different name for your solution.
```

Comment:
Change "You may need to" to "To avoid conflicting with the code from the previous tutorial, we'll give this solution a different name, Hello World Uno"